sdk/lib/crt/math/i386/exp2_asm.s

```diff
@@ -0,0 +1,38 @@
+#include <asm.inc>
```

Comment:
Missing copyright notice?

Reply:
This is a copy of exp code, which is also left without copyright.